### PR TITLE
Added python program for bubble sort

### LINKED ISSUE #404: BubbleSort in Python3
This is the issue about implementation of algorithm of bubble sort in python 3

### PR DESCRIPTION
**Pull request by Aaditya1978**
**Hi!**  
Thanks for considering contributing to this ever-growing list of algorithm and data structure implementations.
Your contribution is valuable. Use the follow template for creating Pull Requests
---

**Fixes #404**

By submitting this pull request I confirm I've read and complied with the below declarations.

-   [x] I have read the [Contribution guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md) 
and I am confident that my PR reflects them.

-   [x] I have followed the [coding guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#cs) 
for this project.

-   [x] My code follows the [skeleton code structure](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#sample).

-   [x] This pull request has a descriptive title. 
For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.

-   [x] This pull request shall only be merged once the build passes.

-   [x] I have mentioned the issue number correctly (with hyper-link) in this pull request description.

### CODE REVIEW

File: algorithms/ar-bsrt/python3/bubble_sort_exe.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    length = int(input('Enter the length of the array to be entered : ')) # Taking the length of array
+    array = [int(i) for i in input('Enter Array Elements : ').split()]    # Taking array elements
+    sorted_array = bubble_sort(length,array)     # Calling the function for sorting the array using bubble sort
+    print("Sorted Array is : ")
+    for i in sorted_array:                       # Printing the sorted array
+        print(i, end = " ")
+
+
+# Running the main code of the program
+if __name__ == '__main__':
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/python3/bubble_sort_exe.py:25]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6280066)

File: algorithms/ar-bsrt/python3/bubble_sort_exe.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        j = 0
+        for j in range(0, length-i-1):
+            if array[j] > array[j+1]:
+                array[j], array[j+1] = array[j+1], array[j]
+
+    return array
+
+
+# This is the main function of the program
+def main():
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/python3/bubble_sort_exe.py:15]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6280066)

File: algorithms/ar-bsrt/python3/bubble_sort_exe.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+def bubble_sort(length, array): # It takes two arguments -> Length of the array and the array itself.
+                                # It returns the sorted array
+    for i in range(length):
+        j = 0
+        for j in range(0, length-i-1):
+            if array[j] > array[j+1]:
+                array[j], array[j+1] = array[j+1], array[j]
+    return array
+# This is the main function of the program
+def main():
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/python3/bubble_sort_exe.py:11]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6280066)

File: algorithms/ar-bsrt/python3/bubble_sort_exe.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+# This is the main function of the program
+def main():                                   
+    length = int(input('Enter the length of the array to be entered : ')) # Taking the length of array
+    array = [int(i) for i in input('Enter Array Elements : ').split()]    # Taking array elements
+    sorted_array = bubble_sort(length,array)     # Calling the function for sorting the array using bubble sort
+    print("Sorted Array is : ")
+    for i in sorted_array:                       # Printing the sorted array
+        print(i, end = " ")
+# Running the main code of the program
+if __name__ == '__main__':
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/python3/bubble_sort_exe.py:19]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6280066)

**Review by RavinduSachintha — APPROVED** [review]

**Comment by RavinduSachintha** [comment]
@Aaditya1978 Nice work. :+1: 
It would be great if you can mark this implementation on the main README file. You just need to copy-paste the phrase for "correct sign" to the right place at the chart. No need to make another PR. just change the file and push it.

**Comment by RavinduSachintha** [comment]
@Aaditya1978 I will merge this PR now. Thank you for your contribution. :smiley: 
@all-contributors please add @Aaditya1978  for code

**Comment by Aaditya1978** [comment]
> @Aaditya1978 Nice work. 👍
> It would be great if you can mark this implementation on the main README file. You just need to copy-paste the phrase for "correct sign" to the right place at the chart. No need to make another PR. just change the file and push it.

In which branch I had to push. Can I push to the branch which is merged.

**Comment by RavinduSachintha** [comment]
> > @Aaditya1978 Nice work. +1
> > It would be great if you can mark this implementation on the main README file. You just need to copy-paste the phrase for "correct sign" to the right place at the chart. No need to make another PR. just change the file and push it.
> 
> In which branch I had to push. Can I push to the branch which is merged.

It is not a mandatory thing. I will update the readme file. Thank you for your contribution and welcome for more contributions.

**Comment by Aaditya1978** [comment]
> > > @Aaditya1978 Nice work. +1
> > > It would be great if you can mark this implementation on the main README file. You just need to copy-paste the phrase for "correct sign" to the right place at the chart. No need to make another PR. just change the file and push it.
> > 
> > 
> > In which branch I had to push. Can I push to the branch which is merged.
> 
> It is not a mandatory thing. I will update the readme file. Thank you for your contribution and welcome for more contributions.

Thanks @RavinduSachintha

**Pull request merged**